transient-aws/hive-example/query.sql

```diff
@@ -0,0 +1,259 @@
+
+create database tpcds_parq;
+use tpcds_parq;
+
+create external table store_sales
+(
+    ss_sold_date_sk           int,
+    ss_sold_time_sk           int,
+    ss_item_sk                int,
+    ss_customer_sk            int,
+    ss_cdemo_sk               int,
+    ss_hdemo_sk               int,
+    ss_addr_sk                int,
+    ss_store_sk               int,
+    ss_promo_sk               int,
+    ss_ticket_number          int,
+    ss_quantity               int,
+    ss_wholesale_cost         float,
+    ss_list_price             float,
+    ss_sales_price            float,
+    ss_ext_discount_amt       float,
+    ss_ext_sales_price        float,
+    ss_ext_wholesale_cost     float,
+    ss_ext_list_price         float,
+    ss_ext_tax                float,
+    ss_coupon_amt             float,
+    ss_net_paid               float,
+    ss_net_paid_inc_tax       float,
+    ss_net_profit             float
+)
+stored as parquet
+location 's3a://cloudera-demo/datasets/tpcds/nonpartitioned/parq/scale_1/store_sales';

```

Comment:
Sounds good. I've pinged folks on the corresponding HD jira to get that
moving.

On Wed, Aug 24, 2016 at 10:51 AM, tdeane <notifications@github.com> wrote:

> In transient-aws/hive-example/query.sql
> <https://github.com/cloudera/director-scripts/pull/12#discussion_r76105754>
> :
>
> > +    ss_quantity               int,
> > +    ss_wholesale_cost         float,
> > +    ss_list_price             float,
> > +    ss_sales_price            float,
> > +    ss_ext_discount_amt       float,
> > +    ss_ext_sales_price        float,
> > +    ss_ext_wholesale_cost     float,
> > +    ss_ext_list_price         float,
> > +    ss_ext_tax                float,
> > +    ss_coupon_amt             float,
> > +    ss_net_paid               float,
> > +    ss_net_paid_inc_tax       float,
> > +    ss_net_profit             float
> > +)
> > +stored as parquet
> > +location 's3a://cloudera-demo/datasets/tpcds/nonpartitioned/parq/scale_1/store_sales';
>
> I was just writing to ask you about this when I saw this comment. Yes, I
> think it's a good idea to wait for CLOUD-3966 before we publish. Once
> that's ready I'll test the script once more and update the pull request.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/cloudera/director-scripts/pull/12/files/1a1bd4368709be8e6eb90b07965e54fc4d770663#r76105754>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAUoXWS7ta69LNpdg5ushgsGLWg-z9NXks5qjISpgaJpZM4Jn3Lu>
> .
>